llvm/test/CodeGen/AMDGPU/stack-realign.ll

```diff
@@ -32,7 +32,6 @@ define void @needs_align16_default_stack_align(i32 %idx) #0 {
 ; GCN-NEXT:    buffer_store_dword v1, v0, s[0:3], 0 offen
 ; GCN-NEXT:    s_waitcnt vmcnt(0)
 ; GCN-NEXT:    s_setpc_b64 s[30:31]
-; GCN: ; ScratchSize: 144
```

Comment:
Lost all of these ScratchSize checks

Reply:
It's unfortunate that they're in an otherwise auto-generated test file. But I'm going to restore them.